### DISCUSSION TITLE
Run makeprg, but show output in ``trouble.nvim`` instead of quickfix list

### DISCUSSION THREAD

**Discussion by wjakob** [stevearc/overseer.nvim#271] [Q&A]
I gave `overseer.nvim` a try to run a lengthy build system as a background task. The introduction helpfully even contains a template to run `makeprg`:
```lua
vim.api.nvim_create_user_command("Make", function(params)
...
```
This is great and worked, but now I am wondering how to slightly deviate from this template.

I stopped using the built-in quickfix list in favor of folke's [trouble.nvim](https://github.com/folke/trouble.nvim) which I find much more ergonomic. How can I modify the `on_output_quickfix` component into an analogous `on_output_trouble`?

Thank you very much!

**Comment by vheon** [comment]
trouble.nvim is a UI for the builtin vim.diagnostics. Just use the overseer's components to fill the diagnostics and Trouble will be populated automatically.
